Formula/vault.rb

```diff
@@ -34,6 +34,39 @@ def install
     end
   end
 
+  plist_options :manual => "valt server -dev"
+
+  def plist; <<~EOS
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <plist version="1.0">
+      <dict>
+        <key>KeepAlive</key>
+        <dict>
+          <key>SuccessfulExit</key>
+          <false/>
+        </dict>
+        <key>Label</key>
+        <string>#{plist_name}</string>
+        <key>ProgramArguments</key>
+        <array>
+          <string>#{opt_bin}/vault</string>
+          <string>server</string>
+          <string>-dev</string>
+        </array>
+        <key>RunAtLoad</key>
+        <true/>
+        <key>WorkingDirectory</key>
+        <string>#{var}</string>
+        <key>StandardErrorPath</key>
+        <string>#{var}/log/vault.log</string>
```

Comment:
This should probably be a separate file.

Reply:
if you wish, all the others are like this however:

https://github.com/Homebrew/homebrew-core/blob/master/Formula/consul.rb
https://github.com/Homebrew/homebrew-core/blob/master/Formula/nomad.rb

Reply:
Unless upstream provides a plist file, then this is fine IMO. There are 10s or hundreds of packages, and as far as I know we don't have a policy not to add inline plists or an effort to migrate them some other format.

Reply:
excellent I was working on separating this out, but this make it much easier to maintain. Thank you